packages/canary/src/components/text.tsx

```diff
@@ -33,7 +33,7 @@ const textVariants = cva('text-base', {
       both: ''
     },
     truncate: {
-      true: 'truncate text-ellipsis text-nowrap'
```

Comment:
this class already contains the required properties, so they aren't necessary

![Google Chrome 2024-11-11 17 05 32](https://github.com/user-attachments/assets/8df0cba1-4ec5-4a8f-86e2-00a1f4308bf5)